src/lua/skiller/subskill_jumpstate.lua

```diff
@@ -228,20 +230,44 @@ function SubSkillJumpState:do_init()
 
    self:skill_reset()
    self.skill_status = skillstati.S_RUNNING
-
+   
    self.args = {}
+
+   -- store expected size of arguments table to compare after run of init()
+   local exp_args = {}
+   local act_args = {}
+
    for _, s in ipairs(self.skills) do
-			if s[1] ~= nil then
-				 local sname = ""
-				 if type(s[1]) == "string" then
-						sname = s[1]
-				 else
-						sname = s[1].name
-				 end
-				 self.args[sname] = {}
-			end
-	 end
+      if s[1] ~= nil then
+         local sname = ""
+         if type(s[1]) == "string" then
+            sname = s[1]
+         else
+            sname = s[1].name
+         end
+         self.args[sname] = {}
+      end
+   end
+   -- copy expected arguments table for later comparison
+   exp_args = table.deepcopy(self.args)
+
    self:init()
+   
+   -- copy actual arguments table for comparison
+   act_args = table.deepcopy(self.args)
+
+   -- compare keys of copies to determine unexpected key names
+   for act_args_key,act_args_val in pairs(act_args) do
+      for exp_args_key,act_args_val in pairs(exp_args) do 
+         if act_args_key == exp_args_key then
+            act_args[act_args_key] = nil
+         end
+      end
+   end
+   for unexp_skillname,_ in pairs(act_args) do
```

Comment:
So looks to me like the `act_args` table ultimately indicates skill arguments that do not belong. In that case I'd prefer calling it `bogus_args` or something similar because that then reflects its true meaning.